lib/Runtime/Language/SourceTextModuleRecord.cpp

```diff
@@ -220,6 +224,38 @@ namespace Js
         return hr;
     }
 
+    HRESULT SourceTextModuleRecord::InstantiateAndNotifyHostAboutDynamicModuleReady()
+    {
+        HRESULT hr = NOERROR;
+        if (!WasDeclarationInitialized())
+        {
+            ScriptContext* scriptContext = GetScriptContext();
+            ModuleDeclarationInstantiation();
+
+            if (this->errorObject != nullptr)
+            {
+                SourceTextModuleRecord::ResolveOrRejectDynamicImportPromise(false, this->GetErrorObject(), scriptContext, this);
```

Comment:
>scriptContext [](start = 107, length = 13)

nitpick : could you not use the this->scriptContext? I am not sure why we need to have local variable for a scriptcontext.